targets/psa_switch/psa_switch.cpp

```diff
@@ -141,23 +144,16 @@ PsaSwitch::receive_(port_t port_num, const char *buffer, int len) {
                                bm::PacketBuffer(len + 512, buffer, len));
 
   BMELOG(packet_in, *packet);
-
   PHV *phv = packet->get_phv();
-  // many current P4 programs assume this
-  // it is also part of the original P4 spec
   phv->reset_metadata();
 
-  // setting standard metadata
-
-  phv->get_field("psa_ingress_parser_input_metadata.ingress_port").set(port_num);
   // TODO
-  // Use appropriate enum member value in JSON
```

Comment:
why are we removing this comment, the TODO is still valid

Reply:
my bad, there's work for this in a separate branch that's not quite done yet, perhaps this should not have been removed just yet.